generate/generate.go

```diff
@@ -21,7 +21,8 @@ var (
 
 // Generator represents a generator for a container spec.
 type Generator struct {
-	spec *rspec.Spec
+	spec         *rspec.Spec
+	hostSpecific bool
```

Comment:
I currently did not think of any cases where `Generator.hostSpsecific` is queried outside the package.